main.css

```diff
@@ -56,9 +53,201 @@ h3 {
   font-size: 26px;
   text-align: center;
   text-decoration: underline;
-  color: darkblue;
+  color: midnightblue;
 }
 a[href] {
   color: black;
   text-decoration: none;
 }
+
+@media screen and (max-width: 1022px) {
+  main {
+    font-size: 22px;
+    width: auto;
```

Comment:
If you don't specify a value for `width` it will be `auto` by default https://developer.mozilla.org/en-US/docs/Web/CSS/width#Formal_definition